### ISSUE TITLE
Custom "define" function triggers a warning about declaring new symbols

### ISSUE THREAD

**Issue by Rudloff** [squizlabs/PHP_CodeSniffer#1769]
Hello,

I have a function called `define()`:
```php
<?php
Foo::define();
```

When I use `phpcs --standard=PSR2`, I get this warning:
```
---------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------
 1 | WARNING | A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should
   |         | execute logic with side effects, but should not do both. The first symbol is defined on line 2 and the first side
   |         | effect is on line 2.
---------------------------------------------------------------------------------------------------------------------------------------
```

I suppose PHPCS mistakes my function for the `define` function included in PHP?

**Comment by gsherwood** [comment]
The sniff wasn't supporting statically called `define` methods, but is now. Thanks for reporting this.
